apps/internal-ui/src/components/Checkbox/types/index.ts

```diff
@@ -0,0 +1,23 @@
+import { InputHTMLAttributes } from 'react';
+
+import { CHECKBOX_SIZES } from '@/components/Checkbox/constants';
+
+export type CheckboxSize = (typeof CHECKBOX_SIZES)[keyof typeof CHECKBOX_SIZES];
+
+export interface CheckboxProps
+  extends Pick<
+    InputHTMLAttributes<HTMLInputElement>,
+    'checked' | 'disabled' | 'onChange' | 'className'
+  > {
+  size?: CheckboxSize;
+  label?: string;
+}
+
+export interface CheckboxIconProps
+  extends Required<Pick<CheckboxProps, 'size'>> {}
+
+export interface CheckboxLabelProps
+  extends Required<Pick<CheckboxProps, 'size'>> {
+  label: string;
+  labelId?: string;
+}
```

Comment:
```suggestion
export interface CheckboxLabelProps
  extends Required<Pick<CheckboxProps, 'size' | 'label'>> {
  labelId?: string;
}
```
p2: label 도 extend 에 추가하면 좋을 것 같습니다!